editoast/Cargo.toml

```diff
@@ -199,6 +199,7 @@ axum = { version = "0.7.9", default-features = false, features = [
 editoast_authz = { workspace = true, features = ["fixtures"] }
 editoast_models = { workspace = true, features = ["testing"] }
 editoast_osrdyne_client = { workspace = true, features = ["mock_client"] }
+opentelemetry_sdk = { version = "0.27.1", features = ["testing"] }
```

Comment:
Since it's defined twice now, maybe we want to define it in the `workspace` dependencies, in order to define the version in only one place?

Reply:
Done.